### ISSUE TITLE
Ivy: Component's class and attribute selectors are missing in created html element when router outlet is used

### ISSUE THREAD

**Issue by yurikolesnikov** [angular/angular#35445]
# 🐞 bug report

### Affected Package
Ivy

### Description
If component has class or attribute selectors they are removed when component is created via routing.

Component:
```typescript
import { Component } from "@angular/core";

@Component({
    selector: "my-component.main[test]",
    template: "My component"
})
export class MyComponent {
}
```

When it's used in another template like this:
```html
<my-component class="main" [test]></my-component>
```
then it's created as expected.
```html
<my-component class="main" test>My component</my-component>
```

When it's created via routing in router-outlet, then class and attribute are missing in resulted html:
```html
<my-component>My component</my-component>
```

There is no such problem when Ivy is not enabled.

## 🔬 Minimal Reproduction

https://stackblitz.com/edit/angular-v2cpwe

## 🌍  Your Environment

**Angular Version:**
<pre><code>
Angular CLI: 9.0.2
Node: 12.14.1
OS: win32 x64

Angular: 9.0.1
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router
Ivy Workspace: Yes

Package                            Version
------------------------------------------------------------
@angular-devkit/architect          0.900.2
@angular-devkit/build-angular      0.900.2
@angular-devkit/build-ng-packagr   0.900.2
@angular-devkit/build-optimizer    0.900.2
@angular-devkit/build-webpack      0.900.2
@angular-devkit/core               9.0.2
@angular-devkit/schematics         9.0.2
@angular/cli                       9.0.2
@ngtools/webpack                   9.0.2
@schematics/angular                9.0.2
@schematics/update                 0.900.2
ng-packagr                         9.0.0
rxjs                               6.5.4
typescript                         3.7.5
webpack                            4.41.2
</code></pre>

**Comment by pkozlowski-opensource** [comment]
I believe that the existing PR #34481 should fix it.

**Comment by AndrewKushnir** [comment]
Hi @yurikolesnikov, the changes from PR #34481 were merged to master and released to NPM as a part of 9.0.2 version. Please try upgrading your app to 9.0.2 version and checking the behavior again. Feel free to reopen this ticket if the problem still exists. Thank you.
